charts/nodejs/templates/ingress.yaml

```diff
@@ -55,10 +55,8 @@ spec:
             port:
               number: {{ .Values.service.externalPort }}
 {{- if .Values.service.additionalHosts }}
```

Comment:
Does this equality work? Will the value of `.Values.service.additionalHosts` be strings or boolean? Because in the UI you're setting the value to "true" or "false", I'm not sure how they're parsed.

Reply:
I am still about to test it.

Reply:
yes, they are correctly parsed and evaluated as booleans